client/profile-wizard/index.js

```diff
@@ -232,23 +234,19 @@ class ProfileWizard extends Component {
 			updateNote( profilerNote.id, { status: 'actioned' } );
 		}
 
-		updateProfileItems( { completed: true } )
-			.then( () => {
-				if ( shouldConnectJetpack ) {
-					document.body.classList.add(
-						'woocommerce-admin-is-loading'
-					);
-				}
-			} )
-			.then( () => {
-				if ( shouldConnectJetpack ) {
-					connectToJetpack(
-						getHistory().push( getNewPath( {}, '/', {} ) )
-					);
-				} else {
-					getHistory().push( getNewPath( {}, '/', {} ) );
-				}
-			} );
+		updateProfileItems( { completed: true } ).then( () => {
+			clearTaskCache();
+
+			if ( shouldConnectJetpack ) {
+				document.body.classList.add( 'woocommerce-admin-is-loading' );
```

Comment:
The extra promise in the chain was redundant here and this is functionally the same as before.